### DISCUSSION TITLE
Contributing: organization membership

### DISCUSSION THREAD

**Discussion by rwp0** [StrawberryPerl/Perl-Dist-Strawberry#110] [General]
Can we get organization membership as users for (or to start) contributing here and there to the project?

**Comment by shawnlaffan** [comment]
There is no need to be a member to file issues or PRs.  That's how I started contributing.  Membership came later.

**Reply by rwp0** [reply]
OK but there are maintenance tasks like [this](https://github.com/StrawberryPerl/Perl-Dist-Strawberry/discussions/112) we cannot do without being a member.

It also shows in the GH profile badge increasing enthusiasm and encouraging contributions.
